java/ql/src/Security/CWE/CWE-352/CsrfUnprotectedRequestType.qhelp

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE qhelp SYSTEM "qhelp.dtd">
+<qhelp>
+
+<overview>
+<p>When you set up a web server to receive a request from a client without any mechanism
+for verifying that it was intentionally sent, then it is vulnerable to attack. An attacker can
```

Comment:
I think in this case it might make sense to be explicit/clearer about it being a CSRF attack.

Reply:
This overview section is directly copied from the [SpringCSRFProtection QHelp](https://github.com/github/codeql/blob/5e927634bc9822749f1bd32c22899805754830bd/java/ql/src/Security/CWE/CWE-352/SpringCSRFProtection.qhelp#L5-L9) since I wanted them to be aligned, but I agree that the wording could be better, so I've updated the wording here to explicitly mention CSRF and will open a small PR to align the Spring QHelp wording.